woof-code/rootfs-skeleton/usr/local/petget/removemodes.sh

```diff
@@ -39,8 +39,7 @@ report_window () {
  
  if [ -s /tmp/petget_proc/overall_petget-deps-maybe-rem ];then
   MAYBEREM="`cat /tmp/petget_proc/overall_petget-deps-maybe-rem | cut -f 1 -d ' ' | tr '\n' ' '`"
-  MAYBEREMMSG1="$(gettext 'The following package(s) are dependencies for the package(s) you just removed. You may want to remove them too or reinstall the package(s) you just removed'):
-<i><b>${MAYBEREM}</b></i>"
+  MAYBEREMMSG1="$(gettext 'The following package(s) are dependencies for the package(s) you just removed. You may want to remove them too or reinstall the package(s) you just removed'): ${MAYBEREM}"
```

Comment:
I see that in this one (and the other 2 dialog related commits) you remove any text highlights.
This one in particular also provides the list of packages in the same line as the information text.
Of course aesthetics are  different and changing with time but would be a good idea to provide side by side images for an immediate visual suggestion.

Sorry for not commenting on the other pulls directly, but I also notice that you block gtkdialog window resizing in some cases, why not. 
Also code indentation  one case in #1462 does not exist. @wdlkmpx will rewrite it!  😈

Reply:
Speaking of code indintation I prefer two spaces rather than tabs but I suppose tabs has one the battle and I'm in the minority with this preference.